### PR TITLE
Replaced sale.shop to sale.channel #7410

### PR DESCRIPTION
**Pull request by rajatguptarg**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -4,8 +4,8 @@ psycopg2
 https://github.com/openlabs/nereid/archive/develop.tar.gz
 https://github.com/openlabs/nereid-catalog/archive/develop.tar.gz
 
-# Install zikzakmedia's sale shop module from bitbucket.
-https://bitbucket.org/zikzakmedia/trytond-sale_shop/get/default.tar.gz
+# Install sale channel module from GitHub
```
Comment (tarunbhardwaj) [inline requirements.txt]: install this from PyPi

File: requirements.txt [hunk 2]
```diff
@@ -4,8 +4,7 @@ psycopg2
 https://github.com/openlabs/nereid/archive/develop.tar.gz
 https://github.com/openlabs/nereid-catalog/archive/develop.tar.gz
 
-# Install zikzakmedia's sale shop module from bitbucket.
-https://bitbucket.org/zikzakmedia/trytond-sale_shop/get/default.tar.gz
-
+# Install sale channel module from PyPI
+openlabs_sale_channel
```
Comment (tarunbhardwaj) [inline requirements.txt]: need not to add here it is already in depends

File: tests/test_product.py [hunk 3]
```diff
@@ -371,15 +371,27 @@ def setup_defaults(self):
...
+                'source': 'manual',
+                'create_users': [('add', [USER])],
                 'warehouse': warehouse,
                 'payment_term': payment_term,
                 'company': self.company.id,
-                'users': [('add', [USER])]
             }])
-        self.User.set_preferences({'shop': self.shop})
+
+        self.User.set_preferences({'channel': self.channel})
```
Comment (tarunbhardwaj) [inline tests/test_product.py]: its current_channel

https://github.com/openlabs/trytond-sale-channel/blob/develop/user.py#L39

File: tests/test_product.py [hunk 4]
```diff
@@ -484,7 +496,7 @@ def test_0020_test_partner_price(self):
                 )
 
     def test_0030_test_guest_price_fallback(self):
-        """
+        """English
```
Comment (tarunbhardwaj) [inline tests/test_product.py]: ?

**Comment by tarunbhardwaj** [comment]
LGTM

File: CHANGELOG [hunk 5]
```diff
@@ -1,25 +1,25 @@
 Version 3.2.1.0
 ===============
 
-* Depend on sale_shop module
+* Depend on sale_channel module
```
Comment (sharoonthomas) [inline CHANGELOG]: WTF 

Write a new CHANGELOG for 3.4

File: CHANGELOG [hunk 6]
```diff
@@ -1,25 +1,22 @@
-Version 3.2.1.0
+Version 3.4.1.1
```
Comment (tarunbhardwaj) [inline CHANGELOG]: DO NOT remove 3.2 changes

File: CHANGELOG [hunk 7]
```diff
@@ -23,3 +23,26 @@ Version 3.2.1.0
 
   Migration: Ensure that you set the pricelist of the shop as the
   same pricelist that was set for the guest user.
+
+Version 3.4.1.1
```
Comment (sharoonthomas) [inline CHANGELOG]: push the new version changelog up.

**Pull request merged**